cmd/flags.go

```diff
@@ -259,3 +261,15 @@ func retryFlag(cmd *cobra.Command) *cobra.Command {
 	}
 	return cmd
 }
+
+func updateTimeFlags(cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().DurationP(flagTimeInterval, "i", 60*time.Second, "time interval to run update client process")
+	cmd.Flags().Float64P(flagThresholdTime, "n", 10, "update client n minutes before to expiry time")
```

Comment:
I think we should be updating well before 10 minutes before expiry time. That's cutting it too close in my opinion. I think something between 6 hrs and 1 day is more safe.